### PR TITLE
Add urls column to replays dataset

### PR DESCRIPTION
**Pull request by cmanallen**
-- start migrations

-- migration replays : 0002_add_url
Local operations:
ALTER TABLE replays_local ADD COLUMN IF NOT EXISTS url String AFTER title;

Dist operations:
ALTER TABLE replays_dist ADD COLUMN IF NOT EXISTS url String AFTER title;
-- end migration replays : 0002_add_url

### CODE REVIEW

File: snuba/datasets/storages/replays.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
             Array(UUID()),
         ),  # TODO: create bloom filter index / materialize column
         ("title", String(Modifiers(readonly=True))),
+        ("urls", Array(String())),
```
Comment (nikhars) [inline snuba/datasets/storages/replays.py:28]: Could you provide additional context on why this column needs to be an array?

Comment (cmanallen) [reply snuba/datasets/storages/replays.py:28]: @nikhars It doesn't necessarily have to be an array.  We could write all the urls as separate replay record.  Is there a benefit to choosing one over the other?

Comment (cmanallen) [reply snuba/datasets/storages/replays.py]: This became a moot point after a requirements change.

**Review by nikhars — APPROVED** [review]

**Comment by evanh** [comment]
Please add the generated SQL of the migration to the PR description.

**Comment by cmanallen** [comment]
@evanh How do I do this?  I don't see anything in the docs.

**Review by JoshFerge — APPROVED** [review]

**Comment by nikhars** [comment]
> @evanh @nikhars Build failing with `ERROR: build step 4 "gcr.io/sentryio/docker-compose" failed: context deadline exceeded`. Thoughts?

Please re-run the failed step and see what happens. Might have been an infrastructure problem.

**Pull request merged**